recipes/git/bld.bat

```diff
@@ -0,0 +1,16 @@
+7za x PortableGit-2.6.4-32-bit.7z.exe -o"%LIBRARY_PREFIX%\" -aoa
+cd %LIBRARY_PREFIX%
+call post-install.bat
+del git_bash.exe
+del git_cmd.exe
+del README.portable
+del post-install.bat
+
+IF NOT EXIST %PREFIX%\Menu mkdir %PREFIX%\Menu
+copy %RECIPE_DIR%\menu-windows.json %PREFIX%\Menu\
+copy %RECIPE_DIR%\git-for-windows.ico %PREFIX%\Menu\
+
+echo export PATH=$(cygpath -a %PREFIX:\=/%)/Library/bin:$PATH >> %LIBRARY_PREFIX%\etc\profile.d\env.sh
+echo. >> %LIBRARY_PREFIX%\etc\profile.d\env.sh
+
+exit 0
```

Comment:
AFAICT building from source will become an option when @mingwandroid gets a bit further on packaging MSYS2 inside conda.  I don't know if anyone has succeeded at building git with a native Windows toolchain (I couldn't find any examples).

The EXEs that are removed are ones that live in the root of that extracted package.  There are equivalents in bin that are not deleted.  There's both git-bash and bash (which I think do the same thing?)

I have used an earlier version of this recipe to teach a SWC class.  It worked very well - especially with the shortcut that this recipe installs.  It actually only had problems on Mac (due to certificates not being found.)  I did fix that - I'll be sure to merge those changes here.  The fix was basically to download certs from cURL's site and put them in the correct place, then tell git where to find them.